RoboticonColony/Assets/Code/Classes/Inventory.cs

```diff
@@ -0,0 +1,103 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+
+/// <summary>
+/// An inventory. Holds all resources for a player or market and facilitates transactions with other inventories.
+/// </summary>
+sealed public class Inventory
```

Comment:
Is it worth adding owned tiles to the inventory so that later on its easy for players to trade tiles?